### PR TITLE
Specifying that you need Helm v2.5.0 for installation

### PR DESCRIPTION
**Pull request by arschles**
This is a follow-up to https://github.com/kubernetes-incubator/service-catalog/pull/1163#discussion_r135920291

Cc/ @MHBauer

### CODE REVIEW

**Comment by duglin** [comment]
LGTM

**Comment by MHBauer** [comment]
@arschles Thanks for following up!

File: docs/install-1.6.md [hunk 1]
```diff
@@ -27,14 +27,14 @@ KUBE_ENABLE_CLUSTER_DNS=true hack/local-up-cluster.sh -O
 
 ## Helm
 
-You *must* use [Helm](http://helm.sh/) v2 or newer in the installation steps
+You *must* use [Helm](http://helm.sh/) v2.5.0 or newer in the installation steps
 below.
 
-If you already have Helm v2 or newer, execute `helm init` (if you haven't
+If you already have Helm v2.5.0 or newer, execute `helm init` (if you haven't
```
Comment (nilebox) [inline docs/install-1.6.md]: Repeating the version 3 times in a single section looks a bit redundant to me, I would leave if only for the first mention and omit the version completely in all other places

File: docs/install-1.6.md [hunk 2]
```diff
@@ -27,14 +27,14 @@ KUBE_ENABLE_CLUSTER_DNS=true hack/local-up-cluster.sh -O
...
 already) to install Tiller (the server-side component of Helm), and you should
 be done with Helm setup.
 
-If you don't already have Helm v2, see the
+If you don't already have Helm v2.5.0, see the
```
Comment (nilebox) [inline docs/install-1.6.md]: "or newer" :)

File: docs/install-1.6.md (hunk 1 above)
Comment (arschles) [reply docs/install-1.6.md]: @nilebox I've removed redundancy

File: docs/install-1.6.md (hunk 2 above)
Comment (arschles) [reply docs/install-1.6.md]: done

**Comment by arschles** [comment]
@nilebox I've addressed your comments. If the changes are acceptable, can you merge (pending CI)?

**Pull request merged**